composer.json

```diff
@@ -18,7 +18,7 @@
     "require-dev": {
         "symfony/config": "2.*",
         "phpunit/phpunit": "3.7.*",
```

Comment:
Happy new year. : I meant if it is necessary at all to define a dependency on PHPUnit.